kernel/base/src/main/java/com/twosigma/beakerx/jvm/object/OutputContainerLayoutManager.java

```diff
@@ -54,6 +55,9 @@ public void setBorderDisplayed(boolean borderDisplayed) {
     if (item == null) {
       return handleNull();
     }
+    if (item instanceof EasyForm) {
```

Comment:
I would suggest create an interface "WidgetItem" with getAsWIdget() method, 
then  EasyForm and Widget should implement the interface.
This will allow us to get rid of "if statement" for Widget and EasyForm.